src/main/java/PlayRace.java

```diff
@@ -0,0 +1,36 @@
+public class PlayRace {
+    private Car cars[];
+    private int playtimes;
+    private int maxposition;
+    private boolean multiwinner;
+    public PlayRace(String [] players, int playtimes){
+        multiwinner = false;
+        maxposition = 0;
+        this.playtimes = playtimes;
+        cars = new Car[players.length];
+        for(int i=0 ; i<players.length ; i++)
```

Comment:
생성자 안에서 많은 로직이 노출되는군요. 메서드로 추출해보면 어떨까요?
```suggestion
        this.cars = initializeCars(players)
```